doc/cephadm/index.rst

```diff
@@ -4,21 +4,32 @@
 Cephadm
 =======
 
-``cephadm`` deploys and manages a Ceph cluster. It does this by connecting the
-manager daemon to hosts via SSH. The manager daemon is able to add, remove, and
-update Ceph containers. ``cephadm`` does not rely on external configuration
-tools such as Ansible, Rook, and Salt.
-
-``cephadm`` manages the full lifecycle of a Ceph cluster. This lifecycle
-starts with the bootstrapping process, when ``cephadm`` creates a tiny
-Ceph cluster on a single node. This cluster consists of one monitor and
-one manager. ``cephadm`` then uses the orchestration interface ("day 2"
-commands) to expand the cluster, adding all hosts and provisioning all
-Ceph daemons and services. Management of this lifecycle can be performed
-either via the Ceph command-line interface (CLI) or via the dashboard (GUI).
-
-``cephadm`` is new in Ceph release v15.2.0 (Octopus) and does not support older
-versions of Ceph.
+``cephadm`` is a utility that can deploy and manage a Ceph cluster. It does
+this by connecting the manager daemon to hosts via SSH. 
+
+Here is a list of some of the things that ``cephadm`` can do:
+
+- ``cephadm`` can add a Ceph container to the cluster.
+- ``cephadm`` can remove a Ceph container from the cluster.
+- ``cephadm`` can update Ceph containers.
+
+``cephadm`` does not rely on external configuration tools like Ansible, Rook, and Salt.
+
+``cephadm`` manages the full lifecycle of a Ceph cluster. This lifecycle starts
+with the bootstrapping process, when ``cephadm`` creates a tiny Ceph cluster on
+a single node. This cluster consists of one monitor and one manager.
+``cephadm`` then uses the orchestration interface (`day 2
+commands <https://www.redhat.com/en/blog/how-does-red-hat-support-day-2-operations>`_,
+(maintenance operations)) to expand the cluster, adding hosts and
```

Comment:
Is this already documented in the existing cephadm documentation?

Reply:
There's this, in the ceph-ansible documentation:

https://docs.ceph.com/projects/ceph-ansible/en/latest/index.html#day-2-operations